.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Did you mean to include this file, or should it be gitignore’d?

Reply:
Yeah it adds to a better developer experience.

Reply:
It's a debug configuration for VSCode.

Reply:
And you even mentioned it in the PR description :-)